hedera-mirror-rest/api/v1/openapi.yml

```
@@ -162,6 +162,32 @@ paths:
           $ref: "#/components/responses/NotFoundError"
       tags:
         - accounts
+  /api/v1/accounts/{senderIdOrAliasOrEvmAddress}/airdrops/outstanding:
+    get:
+      summary: Get get pending airdrops sent by an account
+      description: |
+        Returns pending airdrops that have been sent by an account.
+      operationId: getOutstandingTokenAirdrops
+      parameters:
+        - $ref: "#/components/parameters/accountIdOrAliasOrEvmAddressPathParam"
+        - $ref: "#/components/parameters/limitQueryParam"
+        - $ref: "#/components/parameters/orderQueryParam"
+        - $ref: "#/components/parameters/receiverIdOrAliasOrEvmAddressQueryParam"
+        - $ref: "#/components/parameters/serialNumberQueryParam"
+        - $ref: "#/components/parameters/tokenIdQueryParam"
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TokenAirdropsResponse"
+        400:
+          $ref: "#/components/responses/InvalidParameterError"
+        404:
+          $ref: "#/components/responses/NotFoundError"
+      tags:
+        - accounts
```

Perhaps we should group these two under a separate tag
```suggestion
        - airdrops
```

---

Got it, updated.